flujo/application/runner.py

```
@@ -1319,6 +1319,18 @@ async def resume_async(
+        # Emit a post_run event to allow tracers (e.g., ConsoleTracer) to render the final
+        # completion panel after a resume completes. The initial run already emitted a
+        # post_run reflecting the paused state; this second event represents the true end.
+        try:
+            await self._dispatch_hook(
+                "post_run",
+                pipeline_result=paused_result,
+                context=ctx,
+            )
+        except Exception:
+            # Never let tracing/hooks break control flow
+            pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)